source/intel/asm/crc32c_sse42_asm.c

```diff
@@ -14,15 +14,20 @@
 #        pragma clang diagnostic ignored "-Wdollar-in-identifier-extension"
 #    endif
 
+#ifdef __APPLE__
+#define LABEL(label) "L_" #label "_%="
+#else
+#define LABEL(label) ".L_" #label "_%="
```

Comment:
Why don't we just switch to ALWAYS doing local labels? is there any downside?

Reply:
This is always doing local labels, but helpfully, the local label prefix is different for Mach-O versus ELF.